packages/api/lambdas/sf-starter.js

```diff
@@ -18,18 +18,18 @@ const {
 /**
  * Starts a new stepfunction with the given payload
  *
- * @param {Object} message - incoming queue message
+ * @param {Object} message - incoming SQS message object
  * @returns {Promise} - AWS SF Start Execution response
  */
 function dispatch(message) {
-  const input = Object.assign({}, message.Body);
+  const input = JSON.parse(get(message, 'Body', get(message, 'body', '{}')));
```

Comment:
Under what circumstances would this be `body`?

Reply:
[SQS eventSourceMapping events](https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html#events-sqs-queueconfig) have it named `body`, whereas the output of a receiveMessages call returns records with the field named `Body`.

Reply:
As called out in https://github.com/nasa/cumulus/pull/1402/files#diff-5ba9bc1fb1c5a8d6cb18faa859d06d98R330